daffodil-tdml-processor/src/main/scala/org/apache/daffodil/tdml/TDMLInfosetOutputter.scala

```diff
@@ -47,8 +49,10 @@ class TDMLInfosetOutputter() extends InfosetOutputter {
   private val w3cdomOut = new W3CDOMInfosetOutputter()
   private val jsonOut = new JsonInfosetOutputter(jsonStream, false)
   private val xmlOut = new XMLTextInfosetOutputter(xmlStream, false)
+  private val ch = new SAXHandler()
+  private val saxOut = new SAXInfosetOutputter(ch)
```

Comment:
Not sure we need the SAXInfosetOutputter if we are also duplicating parse using our SAX API in the TDML runner. That shoudl sufficiently tested the infoset outputter.